### ISSUE TITLE
consider adding build instructions

### ISSUE THREAD

**Issue by badair** [SuperV1234/unosolo#2]
Some of us from cpp land are not very familiar with the Rust toolchain.

**Comment by vittorioromeo** [comment]
Good idea, added some in 7ae1e2ad5a1ff9519b33b5ff767b6a5c2b810c59. Let me know if it works for you.

**Comment by badair** [comment]
Thanks, build instructions worked for me.
